src/System.Private.Uri/tests/ExtendedFuctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj

```diff
@@ -20,6 +22,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <TestNugetTargetMoniker Include=".NETStandard,Version=v1.7" />
```

Comment:
This is a recently added project which tests purely new api, so it is intended only to work on netcoreapp1.1 or higher.